### ISSUE TITLE
cleaner method to deploy to github pages

### ISSUE THREAD

**Issue by madjin** [KhronosGroup/glTF-Compressor-Release#1]
I modified the github action so it can deploy from the gh-pages branch in the original project repo, see here: https://github.com/KhronosGroup/glTF-Compressor/pull/21/files#diff-1429285a63936977b05806e8a5aba1ce8e47b48356432b69576b57cb6ef65663

**Comment by allcreatures** [comment]
Do you still need this release repository if hosting on Github pages?

**Comment by madjin** [comment]
> Do you still need this release repository if hosting on Github pages?

no, but maybe khronos group has their reasons? I think it's better if it's all self contained though, can build the demo site on a separate branch such as `gh-pages`

**Comment by allcreatures** [comment]
No, if you do not need it for your purpose, we can delete. Would be good  to ensure we update any links to the release before doing so :)
